site/assets/scss/_anchor.scss

```diff
@@ -1,11 +1,16 @@
-.anchorjs-link {
+.anchor-link {
+  padding-left: .175rem;
   font-weight: 400;
   color: rgba($link-color, .5);
-  @include transition(color .15s ease-in-out);
+  text-decoration: none;
+  opacity: 0;
+  @include transition(color .15s ease-in-out, opacity .15s ease-in-out);
 
+  :hover > &,
```

Comment:
This is an interesting addition. We may also make the whole line clickable using our `.streched-link` helper—not sure it'd be better though, mostly a UX consideration.

Reply:
I just tried to achieve the same behavior as before. Feel free to experiment with your suggestion if you think is better :)